modules/imaging_uploader/jsx/ImagingUploader.js

```diff
@@ -52,7 +53,7 @@ class ImagingUploader extends Component {
         // the StaticDataTable Component.
         loris.hiddenHeaders = this.state.hiddenHeaders;
```

Comment:
```suggestion

```

Reply:
the FIXME comment should also be removed and hiddenHeaders: ['PatientName'] added to line 17